dotnetAvaloniaFrameBuffer/Dockerfile.debug

```diff
@@ -9,7 +9,8 @@ ARG IMAGE_ARCH=
 ##
```

Comment:
Avalonia FrameBuffer template did not work on my side (Verdin IMX8MP). It enters the Debug mode, but after it executes the whole code the GUI does not show on the screen and curiously the device reboots.